### PR TITLE
WALL: Create Radio Component

### PR DESCRIPTION
**Pull request by VasudevaVellanki4**
## Summary

Create Radio component and fixed the dot colors.

### Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (code change that neither fixes a bug nor adds a feature)

### Linked Issues

-NULL

## JIRA Issue(s)

- [JIRA Issue Link](https://wallaceproject.atlassian.net/browse/WALL-12)

## Checklist (check all)

- [ ] Automated tests have been added, or an explanation for why not
- [ ] Security implications and dependencies on linked features have been considered and explained
- NULL

## Screenshots & Evidence

image

_Include any relevant screenshots, Videos, or links to documentation that provide additional context or proof of functionality._

### CODE REVIEW

File: src/components/atoms/RadioComp/RadioComp.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  ) => {
+    const baseStyles =
+      'peer h-4 w-4 shrink-0 rounded-full border ring-offset-background focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:cursor-not-allowed disabled:opacity-50';
+
+    const getBorderColor = () => {
+      if (combinedVariant === 'combined-green-border') return 'border-green-500';
+      if (combinedVariant === 'combined-dark-green-border') return 'border-[#103c34]';
+
+      return variant === 'green-border' ? 'border-green-500' : 'border-[#103c34]';
+    };
```
Comment (raj-upadhyayy) [inline src/components/atoms/RadioComp/RadioComp.tsx:29]: can we try, same for dotcolor too
```
const getBorderColor = () => {
  const greenBorder = 'border-green-500';
  const darkGreenBorder = 'border-[#103c34]';

  if (combinedVariant) {
    return combinedVariant === 'combined-green-border' ? greenBorder : darkGreenBorder;
  }

  return variant === 'green-border' ? greenBorder : darkGreenBorder;
};
```

**Review by raj-upadhyayy — CHANGES_REQUESTED** [review]
Minor change, rest looks good!

**Comment by raj-edition** [comment]
Add summary and update PR title please 
Screenshot 2024-09-12 at 6 14 55 AM

File: src/components/atoms/RadioComp/RadioComp.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    combinedVariant?: RadioCompCombinedVariant;
+    checked?: boolean;
+  }
+>(
+  (
+    { className, variant = 'default', combinedVariant = 'default', checked = false, ...props },
+    ref
+  ) => {
+    const baseStyles =
+      'peer h-4 w-4 shrink-0 rounded-full border ring-offset-background focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:cursor-not-allowed disabled:opacity-50';
```
Comment (raj-edition) [inline src/components/atoms/RadioComp/RadioComp.tsx:22]: lets move this outside RadioComp function

**Review by raj-edition — CHANGES_REQUESTED** [review]
one small change

**Review by raj-edition — APPROVED** [review]

**Review by manikangkan — COMMENTED** [review]
We'll have to revisit this, @VasudevaVellanki4 . I think we can make this better.

CC: @raj-edition

**Comment by manikangkan** [comment]
Merged in https://github.com/wallace-developments/wallace-frontend/pull/30

**Pull request closed without merging**